pkg/recipe/metadata.go

```diff
@@ -72,3 +75,14 @@ func (m *Metadata) Validate() error {
 
 	return nil
 }
+
+func (m *Metadata) Update(re *Recipe, newVer, msg string) error {
```

Comment:
Since the function can not return an error at the moment, the `error` return value can be removed from the signature